Fixes Warning for missing edge ID in TigerXML importer

FlorianZipser opened the pull request.

FlorianZipser committed:
fixes #3

FlorianZipser committed:
fixes #3

amir-zeldes commented:
@FlorianZipser: 
Are you sure you want to merge this into master? Develop is still ahead of master and I understood master updates only happen before a stable release (unless I misunderstood @thomaskrause )

There are also a lot of formatting changes, making this rather hard to review; can you submit a clean version of just the code changes?

FlorianZipser commented:
@amir-zeldes  thanks for checking that. You were right, I didn't want to merge that into master, I wanted to merge it into develop and expected develop to be the default branch. I changed that: First the pull request now merges into develop and second I changed the default branch to develop (so don't be confused).
